libcontainer/seccomp/patchbpf/enosys_linux_test.go

```diff
@@ -105,6 +106,18 @@ var testArches = []string{
 	"ppc64le",
 	"s390",
 	"s390x",
+	// Dummy value to indicate a configuration with no architecture specified.
+	"native",
+}
+
+var nativeArch string
+
+func init() {
+	scmpNativeArch, err := libseccomp.GetNativeArch()
+	if err != nil {
+		logrus.Panicf("get native arch: %v", err)
```

Comment:
Why not just `panic` ?

Reply:
Yeah, I'd rather not add the uneeded logrus dependency (as I hope we'll get rid of it one fine day).

We can use something like this (in a global context)

```go
var nativeArch, _ = libseccomp.GetNativeArch()
```

or, if error checking is needed (I don't know), 

```go
var nativeArch, nativeArchErr = libseccomp.GetNativeArch()

...
func TestNativeArchNoErr(t *testing.T) {
        if nativeArchErr != nil {
                t.Fatalf("libseccomp.GetNativeArch: %v", nativeArchErr)
        }     
}
```

(another idea was to use TestMain but it looks more complicated than it should).

Reply:
Sorry I missed `String()`. 

Something like

```go
var (
     scmpNativeArch, _ = libseccomp.GetNativeArch()
     nativeArch = scmpNativeArch.String()
)
```

should do

Reply:
Not important though

Reply:
I suspect `GetNativeArch` will never fail. I don't really like having an extra global variable and suspicious error checking, but I guess it doesn't matter too much since it's just for testing...